Source/OxyPlot/Series/ContinuousHistogramSeries.cs

```diff
@@ -0,0 +1,560 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="ContinuousHistogramSeries.cs" company="OxyPlot">
+//   Copyright (c) 2014 OxyPlot contributors
+// </copyright>
+// <summary>
+//   Represents an item in a RectangleSeries.
+// </summary>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace OxyPlot.Series
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    using Axes;
+
+    /// <summary>
+    /// Represents a series that can be bound to a collection of <see cref="ContinuousHistogramItem"/>.
+    /// </summary>
+    public class ContinuousHistogramSeries : XYAxisSeries
+    {
+        /// <summary>
+        /// The default tracker format string
+        /// </summary>
+        public new const string DefaultTrackerFormatString = "Start: {5}\nEnd: {6}\nValue: {7}\nArea: {8}";
+        
+        /// <summary>
+        /// The default fill color.
+        /// </summary>
+        private OxyColor defaultFillColor;
+
+        /// <summary>
+        /// The items originating from the items source.
+        /// </summary>
+        private List<ContinuousHistogramItem> actualItems;
+
+        /// <summary>
+        /// Specifies if the <see cref="actualItems" /> list can be modified.
+        /// </summary>
+        private bool ownsActualItems;
+        
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ContinuousHistogramSeries" /> class.
+        /// </summary>
+        public ContinuousHistogramSeries()
+        {
+            this.FillColor = OxyColors.Automatic;
+            this.NegativeFillColor = OxyColors.Undefined;
+            this.StrokeColor = OxyColors.Black;
+            this.StrokeThickness = 0;
+            this.TrackerFormatString = DefaultTrackerFormatString;
+            this.LabelFormatString = null;
+            this.LabelFontSize = 0;
+            this.LabelPlacement = LabelPlacement.Outside;
+        }
+        
+        /// <summary>
+        /// Gets or sets the color of the interior of the bars.
+        /// </summary>
+        /// <value>The color.</value>
+        public OxyColor FillColor { get; set; }
+
+        /// <summary>
+        /// Gets the actual fill color.
+        /// </summary>
+        /// <value>The actual color.</value>
+        public OxyColor ActualFillColor
+        {
+            get { return this.FillColor.GetActualColor(this.defaultFillColor); }
+        }
+        
+        /// <summary>
+        /// Gets or sets the color of the interior of the bars when the value is negative.
```

Comment:
can histograms contain negative values?

Reply:
No, I don't think so, but I felt like providing a half-meaningful behaviour rather than crashing on 'invalid' data. While a 'real' histogram won't contain negatives, I don't see much harm in facilitating a 'dodgy' histogram. I'd be inclined to leave this in, but if I'm missing/underestimating a good reason to remove it I shall do so without complaint.